README.md

```diff
@@ -191,6 +191,7 @@ The Pearl dependencies are the following:
 - [git (>=1.8)](https://git-scm.com/)
 - [GNU coreutils](https://www.gnu.org/software/coreutils/)
 - [grep](https://www.gnu.org/software/grep/)
+- [wget](http://www.gnu.org/software/wget/wget.html) or [cURL](https://curl.haxx.se)
```

Comment:
This line is actually misleading here. Pearl code does not depend on `wget` or `curl`. One of such commands is required only during the Pearl installation phase. I believe the sections below (OSX and Linux) are enough to show what people need to download.

People can even download the script from browser if they want but it is not necessary to list all possible ways ;)

Reply:
true - will remove from that section.